src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,476 @@
+    type: group
+    short-summary: Manage Azure Service Bus Queue AuthorizationRule
+
+    """
+
+helps['servicebus topic'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Topic and authorizationrule
+
+    """
+
+helps['servicebus topic authorizationrule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Topic AuthorizationRule
+"""
+
+helps['servicebus subscription'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Subscription
+"""
+
+helps['servicebus rule'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Rule
+"""
+
+helps['servicebus georecovery-alias'] = """
+    type: group
+    short-summary: Manage Azure Service Bus Geo Disaster Recovery Configuration - Alias
+"""
+
+helps['servicebus namespace exists'] = """
+    type: command
+    short-summary: check for the availability of the given name for the Namespace
+    examples:
+        - name: check for the availability of mynamespace for the Namespace
+          text: az servicebus namespace exists --name mynamespace
+"""
+
+helps['servicebus namespace create'] = """
+    type: command
+    short-summary: Creates a Service Bus Namespace
+    examples:
+        - name: Create a Service Bus Namespace.
+          text: az servicebus namespace create --resource-group myresourcegroup --name mynamespace --location westus
+           --tags ['tag1' 'value1', 'tag2' 'value2'] --sku-name Standard --sku-tier Standard
+"""
+
+helps['servicebus namespace show'] = """
+    type: command
+    short-summary: Shows the Service Bus Namespace details
+    examples:
+        - name: shows the Namespace details.
+          text: helps['az servicebus namespace show --resource-group myresourcegroup --name mynamespace']
+"""
+
+helps['servicebus namespace list'] = """
+    type: command
+    short-summary: List the Service Bus Namespaces by ResourceGroup or by subscription
+    examples:
+        - name: Get the Service Bus Namespaces by resource Group.
+          text: helps['az servicebus namespace list --resource-group myresourcegroup']
+        - name: Get the Service Bus Namespaces by Subscription.
+          text: az servicebus namespace list
+"""
+
+helps['servicebus namespace delete'] = """
+    type: command
+    short-summary: Deletes the Service Bus Namespace
+    examples:
+        - name: Deletes the Service Bus Namespace
+          text: az servicebus namespace delete --resource-group myresourcegroup --name mynamespace
+"""
+
+helps['servicebus namespace authorizationrule create'] = """
+    type: command
+    short-summary: Creates Authorizationrule for the given Service Bus Namespace
+    examples:
+        - name: Creates Authorizationrule 'myauthorule' for the given Service Bus Namespace 'mynamepsace' in resourcegroup
+          text: az servicebus namespace authorizationrule create --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule --access-rights [Send, Listen]
```

Comment:
This won't work. If you want both Send and Listen it should be `--access-rights Send Listen` I guess...

Reply:
Resolved